Gem/Receive/ViewModels/ReceiveViewModel.swift

```diff
@@ -38,7 +42,26 @@ struct ReceiveViewModel {
         )
     }
     
+    func activityItems(qrImage: UIImage?) -> [Any] {
+        let textItems = [Localized.Receive.myAddress(assetModel.networkFullName, assetModel.symbol), address]
+        if let qrImage {
+            return [qrImage] + textItems
+        }
+        return textItems
```

Comment:
```suggestion
        return [qrImage, address].compactMap { $0 }
```

no need to bring the whole text